### PR TITLE
new categories model #6

### PR DESCRIPTION
**Pull request by lotust**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: server/db/models/categories.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Categories = db.define('categories', {
```
Comment (lgrees) [inline server/db/models/categories.js]:
```suggestion
const Category = db.define('category', {
```

File: server/db/models/categories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: { notEmpty: true },
+  }
+})
+
+module.exports = Categories
```
Comment (lgrees) [inline server/db/models/categories.js]:
```suggestion
module.exports = Category
```

File: server/db/models/categories.js (hunk 1 above)
Comment (lgrees) [reply server/db/models/categories.js]: very minor just following the convention they set out with the user model

**Comment by lotust** [comment]
changes submitted to match user model naming convention

**Pull request merged**